File: configlibrary/src/main/java/com/shmedo/configlibrary/iot/model/das/DasSubSensorStatusInfo.java
package com.shmedo.configlibrary.iot.model.das;

/**
 * 创建者:   gonghe <br/>
 * 创建时间:  2021/4/15 <br/>
 * 描述：      DAS 状态页面辅传感器状态
 */
public class DasSubSensorStatusInfo {
    private IoBean io;
    private VwpBean vwp;
    private MemsBean mems;

    public IoBean getIo() {
        return io;
    }

    public void setIo(IoBean io) {
        this.io = io;
    }

    public VwpBean getVwp() {
        return vwp;
    }

    public void setVwp(VwpBean vwp) {
        this.vwp = vwp;
    }

    public MemsBean getMems() {
        return mems;
    }

    public void setMems(MemsBean mems) {
        this.mems = mems;
    }
}

File: lib_cmd/src/main/java/com/shmedo/lib/cmd/base/iot_cmd/model/das/DasSubSensorStatusInfo.kt
package com.shmedo.lib.cmd.base.iot_cmd.model.das

import com.squareup.moshi.JsonClass

/**
 * 创建者:   gonghe <br></br>
 * 创建时间:  2021/4/15 <br></br>
 * 描述：      DAS 状态页面辅传感器状态
 */
@JsonClass(generateAdapter = true)
data class DasSubSensorStatusInfo(
    val io: IoBean? = null,
    var vwp: VwpBean? = null,
    var mems: MemsBean? = null,
)